projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -128,7 +128,11 @@ const IssueFronts = ({
     const { width } = useDimensions()
     /* setting a key will force a rerender on rotation, removing 1000s of layout bugs */
     return (
-        <ScrollView style={style} key={width}>
+        <ScrollView
+            style={[style, { overflow: 'hidden' }]}
+            key={width}
+            removeClippedSubviews={true}
```

Comment:
I think this might break rendering a previously loaded front from the issue screen